.github/workflows/tag-release.yml

```diff
@@ -34,10 +32,20 @@ jobs:
         with:
           # We need to fetch all tags to see if the tag already exists.
           fetch-depth: 0
+      # Check if version/VERSION was changed
```

Comment:
I feel like we could probably put this into the script (if it's not already there), but ... not a blocker